docs/development/manual_tests.md

```diff
@@ -51,9 +54,32 @@ Tool Specific combinations
 - fastp
   - with/without complexity filtering ✅
 - AdapterRemoval
+
   - with/without skipqualitytim ✅
   - with/without skipntrimming ✅
 
+- Markduplicates
+
+  - With FastP
+    - SE&PE data ✅
+    - SE&PE data + preprocessing_excludeunmerged ✅
+    - PE_only + preprocessing_excludeunmerged ✅
+  - With AdapterRemoval
+    - With FastP
+    - SE&PE data ✅
+    - SE&PE data + preprocessing_excludeunmerged ✅
+    - PE_only + preprocessing_excludeunmerged ✅
+
+- Dedup
+  - With FastP
+    - SE&PE data ✅ (expected failure)
+    - SE&PE data + preprocessing_excludeunmerged ✅ (expected failure)
+    - PE_only + preprocessing_excludeunmerged ✅
+  - With AdapterRemoval
+    - SE&PE data ✅ (expected failure)
+    - SE&PE data + preprocessing_excludeunmerged ✅ (expected failure)
+    - PE_only + preprocessing_excludeunmerged ✅
```

Comment:
Ok this stuff probably not necessary as it was just tracking the tests I had done - but I'll remove that on the filter PR once we've merged this in.